lib/gis/parser_json.c

```diff
@@ -340,19 +340,27 @@ char *G__json(void)
  */
 void check_create_import_opts(struct Option *opt, char *element, FILE *fp)
 {
-    int i = 0;
+    int i = 0, j = 0;
     int has_import = 0;
-    char **tokens;
+    char **tokens, **urltokens;
 
+    G_debug(3,"tokenize string: <%s>", opt->answer);
     tokens = G_tokenize(opt->answer, "@");
     while (tokens[i]) {
         G_chop(tokens[i]);
         i++;
     }
```

Comment:
Check if there are more than two tokens, if yes, fatal error. Or split at the first occurrence of "@".

Reply:
Thanks for the idea, added in b601c83